packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.resolver.ts

```diff
@@ -22,6 +33,20 @@ export class ObjectMetadataResolver {
     private readonly beforeUpdateOneObject: BeforeUpdateOneObject<UpdateObjectPayload>,
   ) {}
 
+  @ResolveField(() => String, { nullable: true })
+  async labelPlural(
```

Comment:
I initially wanted to add a middleware on the field decorator which would have been more elegant but it seems like nestjs-query completely ignores it. When we move out of nestjs-query we should use this instead